plc4j/drivers/opcua/src/test/resources/logback-test.xml

```diff
@@ -27,7 +27,9 @@
     </encoder>
   </appender>
 
-  <root level="info">
```

Comment:
check if we set this to info again before we merge.

Reply:
Updated to error.